src/editor-sidebar/metadata-editor-modal.js

```diff
@@ -39,12 +40,25 @@ const ALLOWED_SCREENSHOT_MEDIA_TYPES = [
 	'image/avif',
 ];
 
+const WP_MINIMUM_VERSIONS = [
+	'5.9',
+	'6.0',
+	'6.1',
+	'6.2',
+	'6.3',
+	'6.4',
+	'6.5',
+	'6.6',
+	'6.7',
+];
```

Comment:
This seems tedious to maintain. How about adding `global $wp_version` from PHP into a JS global and using that to generate a range of versions?

Reply:
@creativecoder do you mean something like this? https://github.com/WordPress/create-block-theme/pull/715/commits/f3d352a0d0fd72fa9502368fda8655c0b3d94f30

Reply:
Yes, that's in line with what I was thinking!